src/Files.App/Views/Properties/MainPropertiesPage.xaml

```diff
@@ -55,7 +55,7 @@
 				Height="32"
 				Margin="4,4,0,0"
 				Background="Transparent"
-				BorderBrush="Transparent"
+				BorderThickness="0"
```

Comment:
Shouldn't the border be visible on mouse hover?

Reply:
Fixed